tools/iotools.go

```diff
@@ -89,3 +93,42 @@ func (r *RetriableReader) Read(b []byte) (int, error) {
 
 	return n, errors.NewRetriableError(err)
 }
+
+// Spool spools the contents from 'from' to 'to' by buffering the entire
+// contents of 'from' into a temprorary file, first.
+//
+// A name of "lfs-<time>" is used, where "<time>" is the number of seconds since
+// the Unix epoch.
+//
+// The temporary file is cleaned up after the copy is complete.
+//
+// The number of bytes written to "to", as well as any error encountered are
+// returned.
+func Spool(to io.Writer, from io.Reader) (n int64, err error) {
+	return SpoolName(to, from, fmt.Sprintf("lfs-%d", time.Now().Unix()))
+}
+
+// Spool spools the contents from 'from' to 'to' by buffering the entire
+// contents of 'from' into a temprorary file named "name", first.
```

Comment:
minor nit: the temporary file is _prefixed_ with "name", right? Because `lfs-1486823611` could collide if two malformed files would be processed within one second. Maybe we should rename `name` to `prefix`?